pkgs/development/python-modules/pyvcf/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, pytest
+}:
+
+buildPythonPackage rec {
+
+  pname = "PyVCF";
+  version = "0.6.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    extension = "tar.gz";
+    sha256 = "1ngryr12d3izmhmwplc46xhyj9i7yhrpm90xnsd2578p7m8p5n79";
+  };
+
+  doCheck = false;
```

Comment:
Please leave a comment why tests are disabled

Reply:
The tests needed some missing test files. They are only on the github repository, so I switched to that instead of using PyPi.